bin/new-relic/install.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+command -v git >/dev/null 2>&1 || {
```

Comment:
`hash <program>` is a cleaner, better way to test if an executable is available.

Reply:
Both are qualitative statements which I’d like to quantify here with pointing to the source code of both: https://github.com/bminor/bash/blob/master/builtins/command.def vs https://github.com/bminor/bash/blob/master/builtins/hash.def

Is this the case of the arg being used in cleaner?   Better: is the hashing of a previously traversed PATH location vs full PATH traversal on command.def?
I’m just trying to make the internal case for either/or here